packages/material-ui/src/Popper/Popper.js

```diff
@@ -171,6 +172,7 @@ class Popper extends React.Component {
     return (
       <Portal onRendered={this.handleOpen} disablePortal={disablePortal} container={container}>
         <div
+          ref={this.tooltip}
```

Comment:
I didn't think it would be that easy but it seems like `findDOMNode(this)` drills through portals. Confirmed this by setting a breakpoint in `handleOpen` and the attributes matched the props that were spread from `<Popper {...spread} />`.